### PR TITLE
Code style/merge

### PR DESCRIPTION
**Pull request by Borewit**
Merge and initial review of the following pull requests: 
1. webtorrent/webtorrent-desktop#1262
2. webtorrent/webtorrent-desktop#1263
3. webtorrent/webtorrent-desktop#1266

Some changes of the original pull requests have been taken out.

### CODE REVIEW

**Comment by feross** [comment]
Any reason you made this a separate PR? It's usually desirable to merge the original PR since that's where all the discussion/context exists and it's nice for the PR submitter to get credit for their work (especially if they're a new contributor, since getting a merged PR usually makes them really happy)

File: src/main/windows/main.js [hunk 1]
```diff
@@ -143,23 +143,15 @@ function setBounds (bounds, maximize) {
...
-    willBeMaximized = main.win.isMaximized()
+  if (maximize === true && !main.win.isMaximized()) {
+    log('setBounds: maximizing')
+    main.win.maximize()
+  } else if (maximize === false && main.win.isMaximized()) {
+    log('setBounds: unmaximizing')
+    main.win.unmaximize()
   }
 
+  const willBeMaximized = typeof maximize === 'boolean' ? maximize : main.win.isMaximized
```
Comment (feross) [inline src/main/windows/main.js]: Just realized that this has a bug and `main.win.isMaximized` needs to be invoked as a function. `main.win.isMaximized()`

**Comment by feross** [comment]
Can we get one more review before merge? I'd like us to switch to a model of multiple reviews before merge.

**Comment by michaelgeorgeattard** [comment]
I will update the PRs I created this weekend if you close this.

**Comment by feross** [comment]
@michaelgeorgeattard Sure thing. Happy to wait until the end of the weekend to give you a chance to fix up your original PRs, then we really should move on. Sound fair?

**Comment by Borewit** [comment]
> Any reason you made this a separate PR?

1. I didn't now I could add commits to the original PR, so I will figure out how to do that. Which branch should I commit to? Because I have no access to cloned repo where the pull request is based on. 
2. To combine the three individual PR's and be able to test and review them at once against the current master.

Apparently this is not most efficient way of merging, but I have done in this way more often and as far as i know the original credits are maintained.

**Comment by feross** [comment]
@Borewit Here's some info about how to commit to the requester's branch: https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/ The PR submitter gives permission when they send the PR (there's a checkbox that's checked by default to allow this)

Perhaps you're right about the credit. At this point, I say we give @michaelgeorgeattard until the end of the weekend to update their PRs.

**Comment by Borewit** [comment]
Also the originating PR will be closed and merged automatically once this one is . So do not worry  @michaelgeorgeattard, this was and is an attempt to include your contribution, nothing less.

@feross thanks you, lesson learned, I understand it now.

**Comment by Borewit** [comment]
@michaelgeorgeattard: I pushed my changes back to the originating branches, so you don't have to re-do that work.

**Comment by DiegoRBaquero** [comment]
Are we good to merge this and make a new release?

**Pull request merged**